### PR TITLE
Perl4corelib

### PR DESCRIPTION
**Pull request by pmiess**
A dependency for gnuwin32 giflib command gifburst to work in modern Perl
The icons are my own drawing

support for https://github.com/0install/0install.de-feeds/issues/3

### CODE REVIEW

File: perl/Perl4-CoreLibs.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+distribution is deprecated; they should be acquired from this CPAN
+distribution instead.  From core version 5.13, until their removal,
+it is planned that the core versions of these libraries will emit a
+warning when loaded.  The CPAN version will not emit such a warning.
+</description>
+  <icon href="http://repo.roscidus.com/perl/perl.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="http://repo.roscidus.com/perl/perl.png" type="image/png"/>
+  <category>Development</category>
+  <homepage>https://metacpan.org/pod/Perl4::CoreLibs</homepage>
+  <needs-terminal/>
```
Comment (bastianeicher) [inline perl/Perl4-CoreLibs.xml]: I would suggest removing `<needs-terminal/>` since it only has a meaning for executable feeds (containing `<command>s`) and not for pure library use.

File: perl/Perl4-CoreLibs.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <implementation id="sha1new=b36e6483c2da3e2d1cc6913120ed9c2afa5dbaca" license="Perl License" released="2017-07-29" version="0.004">
+    <archive href="https://cpan.metacpan.org/authors/id/Z/ZE/ZEFRAM/Perl4-CoreLibs-0.004.tar.gz" size="53219" type="application/x-compressed-tar" extract="Perl4-CoreLibs-0.004"/>
+    <archive href="ftp://ftp.cpan.org/pub/CPAN/modules/by-authors/id/Z/ZE/ZEFRAM/Perl4-CoreLibs-0.004.tar.gz" size="53219" type="application/x-compressed-tar" extract="Perl4-CoreLibs-0.004"/>
+    <archive href="http://search.cpan.org/CPAN/authors/id/Z/ZE/ZEFRAM/Perl4-CoreLibs-0.004.tar.gz" size="53219" type="application/x-compressed-tar" extract="Perl4-CoreLibs-0.004"/>
```
Comment (bastianeicher) [inline perl/Perl4-CoreLibs.xml]: Please note that 0install currently does not automatically try the next `<archive>` in the list if the first one fails. It will always pick the first `<archive>` with a MIME type it understands. But this may change in future, so it should be fine to leave the additional `<archive>`s in the feed.

File: perl/Perl4-CoreLibs.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="http://repo.roscidus.com/perl/Perl4-CoreLibs" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
```
Comment (bastianeicher) [inline perl/Perl4-CoreLibs.xml]: The feeds on `repo.roscidus.com` use all-lowercase URIs so far. Probably better to adjust this one for consistency.

File: perl/Perl4-CoreLibs.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  <name>Perl4-CoreLibs</name>
+  <summary xml:lang="en"> libraries historically supplied with Perl 4</summary>
```
Comment (bastianeicher) [inline perl/Perl4-CoreLibs.xml]: Tiny nitpick: Unnecessary whitespace after `>`.

File: perl/Perl4-CoreLibs.xml (hunk 1 above)
Comment (pmiess) [reply perl/Perl4-CoreLibs.xml]: removed

File: perl/Perl4-CoreLibs.xml (hunk 2 above)
Comment (pmiess) [reply perl/Perl4-CoreLibs.xml]: I could have sworn it did that already, hopefully it will sometime soon.

File: perl/Perl4-CoreLibs.xml (hunk 3 above)
Comment (pmiess) [reply perl/Perl4-CoreLibs.xml]: done

File: perl/Perl4-CoreLibs.xml (hunk 4 above)
Comment (pmiess) [reply perl/Perl4-CoreLibs.xml]: removed
